bitnami/grafana-operator/Chart.yaml

```
@@ -25,4 +25,4 @@ name: grafana-operator
 sources:
   - https://github.com/grafana-operator/grafana-operator
   - https://github.com/bitnami/bitnami-docker-grafana-operator
-version: 1.4.0
+version: 1.4.1
```

Since it's a feature addition instead of bug fixing, can you please bump the minor version?
```suggestion
version: 1.5.0
```